README.md

```diff
@@ -474,6 +475,59 @@ session creation time (i.e. when the `SELECT` query is initiated).
 
 Note that this consistency model currently only applies to the table data, not its metadata.
 
+## Spark SQL integration
+
+Dataproc uses a patched version of Spark that automatically detects a table that has the `bq.table`
+table property, in which case Spark will use the [`Spark-BigQuery Connector`](https://github.com/GoogleCloudDataproc/spark-bigquery-connector)
+to access the table's data. This means that on Dataproc you actually do not need to use the
+Hive-BigQuery Connector for Spark SQL.
+
+### Code samples
+
+Java example:
+
+```java
+SparkConf sparkConf = new SparkConf().setMaster("local");
+SparkSession spark =
+    SparkSession.builder()
+    .appName("example")
+    .config(sparkConf)
+    .enableHiveSupport()
+    .getOrCreate();
+Dataset<Row> ds = spark.sql("SELECT * FROM mytable");
+Row[] rows = ds.collect();
+```
+
+Python example:
+
+```python
+spark = SparkSession.builder \
+    .appName("example") \
+    .config("spark.master", "local") \
+    .enableHiveSupport() \
+    .getOrCreate()
+df = spark.sql("SELECT * FROM mytable")
+rows = df.collect()
+```
+
+## Apache Pig integration
```

Comment:
On Dataproc it should normally be as simple as putting the JAR in the directory where the other connectors (Spark, GCS) are. It think it's `/usr/local/share/google/dataproc/lib`?

Reply:
That's right, the user would have to use, for example, the `connectors` init action. However, since the version you're testing isn't released yet, you'd have to install it manually. Maybe could try testing with a single node cluster now, so you just need to copy it to a single node's `/usr/local/share/google/dataproc/lib` directory?